docs/production/ingesters-with-wal.md

```diff
@@ -65,13 +65,12 @@ PS: Given you have to scale down 1 ingester at a time, you can pipeline the shut
 
 **Fallback option**
 
-There is a [flush mode ingester](https://github.com/cortexproject/cortex/pull/1747) in progress, and with recent discussions there will be a separate target called flusher in it's place.
+There is a `flusher` target that can be used to flush the data in the WAL. It's config can be found [here](https://cortexmetrics.io/docs/configuration/) with a `flusher` prefix. As flusher depends on the chunk store and the http API components, you need to also set their config similar to ingesters.
```

Comment:
Please reference https://cortexmetrics.io/docs/configuration/configuration-file/#flusher-config directly. Also the chunk store and http API component docs directly instead of just mentioning.

Reply:
Done. Added a shortcut too for config.